unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/BuilderInWorldBridge/BuilderInWorldBridge.cs

```diff
@@ -151,15 +162,16 @@ void SendNewEntityToKernel(string sceneId, string entityId, ComponentPayload[] c
 
         WebInterface.SceneEvent<AddEntityEvent> sceneEvent = new WebInterface.SceneEvent<AddEntityEvent>();
         sceneEvent.sceneId = sceneId;
-        sceneEvent.eventType = "stateEvent";
+        sceneEvent.eventType = BuilderInWorldSettings.STATE_EVENT_NAME;
         sceneEvent.payload = addEntityEvent;
 
 
-        string messasage = Newtonsoft.Json.JsonConvert.SerializeObject(sceneEvent, Formatting.None, new JsonSerializerSettings
+        //Note (Adrian): We use Newtonsoft instead of JsonUtility because we need to deal with super classes, JsonUtility doesn't enconde them
```

Comment:
```suggestion
        //Note (Adrian): We use Newtonsoft instead of JsonUtility because we need to deal with super classes, JsonUtility doesn't encode them
```